PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/secure/IdeaSecureStore.java

```diff
@@ -20,28 +20,34 @@
  * SOFTWARE.
  */
 
-package com.microsoft.intellij.secure
+package com.microsoft.intellij.secure;
 
-import com.intellij.credentialStore.CredentialAttributes
-import com.intellij.ide.passwordSafe.PasswordSafe
-import com.microsoft.azuretools.securestore.SecureStore
+import com.intellij.credentialStore.CredentialAttributes;
+import com.intellij.ide.passwordSafe.PasswordSafe;
+import com.microsoft.azuretools.azurecommons.helpers.NotNull;
+import com.microsoft.azuretools.azurecommons.helpers.Nullable;
+import com.microsoft.azuretools.securestore.SecureStore;
 
-class IdeaSecureStore : SecureStore {
+public class IdeaSecureStore implements SecureStore {
```

Comment:
better to make the constructor private if it's a singleton.

Reply:
Thanks! Fix it!